src/Nethermind/Nethermind.Merge.Plugin/PostMergeHeaderValidator.cs

```diff
@@ -60,15 +62,27 @@ private bool ValidateTheMergeChecks(BlockHeader header)
             bool validDifficulty =
                 ValidateHeaderField(header, header.Difficulty, UInt256.Zero, nameof(header.Difficulty));
             bool validNonce = ValidateHeaderField(header, header.Nonce, 0u, nameof(header.Nonce));
-            // validExtraData needed in previous version of EIP-3675 specification
-            //bool validExtraData = ValidateHeaderField<byte>(header, header.ExtraData, Array.Empty<byte>(), nameof(header.ExtraData));
             bool validUncles = ValidateHeaderField(header, header.UnclesHash, Keccak.OfAnEmptySequenceRlp,
                 nameof(header.UnclesHash));
+            bool validMixedHash = ValidateHeaderField(header, header.MixHash, Keccak.Zero, nameof(header.MixHash));
 
             return validDifficulty
                    && validNonce
-                   //&& validExtraData
-                   && validUncles;
+                   && validUncles
+                   && validMixedHash;
```

Comment:
mixedHash is wrong here. It is a bit confusing, but please check EIP-4399 https://hackmd.io/@n0ble/kintsugi-spec#EIP-4399